src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -20,18 +20,7 @@ namespace Microsoft.CodeAnalysis.Remote;
 /// </summary>
 internal abstract class RemoteHostClient : IDisposable
 {
-    public event EventHandler<bool>? StatusChanged;
```

Comment:
no one was subscribed to this event.  so i removed it entirely.  simplifying things from there.